### PR TITLE
GH1278: Removed error code for alerts

### PR DESCRIPTION
**Pull request by richsalz**
Commit aea145e removed some error codes that are generated
algorithmically: mapping alerts to error texts.  Found by
Andreas Karlsson.  This restores them, and adds two missing ones.

### CODE REVIEW

**Comment by kroeckx** [comment]
So this is just make errors?

**Comment by richsalz** [comment]
No.  The previous MR used an automated script and removed unused error and function codes.
It did not take into account that some of them (the alerts) are mapped into numbers automatically.

**Comment by kroeckx** [comment]
So can we prevent this from happening again?

**Comment by richsalz** [comment]
I added a test to find-unused-errs to warn about it.  Updated commit pushed.

File: include/openssl/ssl.h [hunk 1]
```diff
@@ -2445,7 +2474,7 @@ void ERR_load_SSL_strings(void);
 # define SSL_R_WRONG_SSL_VERSION                          266
 # define SSL_R_WRONG_VERSION_NUMBER                       267
 # define SSL_R_X509_LIB                                   268
-# define SSL_R_X509_VERIFICATION_SETUP_PROBLEMS           269
+# define SSL_R_X509_VERIFICATION_SETUP_PROBLEMS           101
```
Comment (mattcaswell) [inline include/openssl/ssl.h]: What's this all about? It doesn't seem related to the other changes. Also this reason code has the original value in 1.0.2, so changing it doesn't seem like a good idea.

Comment (richsalz) [inline include/openssl/ssl.h]: I have no idea about the black-magic of mkerrs.  And who cares if the value  changes compared to 1.0.2?

Comment (mattcaswell) [inline include/openssl/ssl.h]: It makes it more difficult to cherry-pick to 1.0.2 from master if there are lots of these changes - and there is no benefit to this one. It should be removed.

Comment (richsalz) [inline include/openssl/ssl.h]: Fine, fixed.

This shows a problem with mkerrs.  You can't rewrite the header file unless an EXISTING error code is removed and then recreated.

**Comment by mattcaswell** [comment]
+1
